common/app/common/controllers/SunriseController.java

```diff
@@ -156,21 +156,21 @@ private static Locale currentLocale(final String languageTag, final ProjectConte
         return projectContext.isLocaleAccepted(locale) ? locale : projectContext.defaultLocale();
     }
 
-    private static CountryCode currentCountry(final Http.Session session, final ProjectContext projectContext) {
+    public static CountryCode currentCountry(final Http.Session session, final ProjectContext projectContext) {
```

Comment:
Yes, I saw that this could be moved, but one step at a time, now we need to bring the request scoped dependency injection into live then we can move a lot of stuff around.